src/components/pluginManager.js

```diff
@@ -34,7 +34,8 @@ define(['events'], function (events) {
 
             require([url, 'globalize', 'appRouter'], function (pluginFactory, globalize, appRouter) {
 
-                var plugin = new pluginFactory();
+                // Because of es6 default exports
+                var plugin = pluginFactory.default ? new pluginFactory.default() : new pluginFactory();
```

Comment:
maybe a little trick?
```js
pluginFactory = pluginFactory.default || pluginFactory;
```

Reply:
Well kinda does the same so I choose that because it looked more concise

Reply:
In my case, you will get separate hack/fix/polyfill and leave other code untouched.

Reply:
Well after terser I'll probably the same anyway I just find mine more readable and that's exactly what we need right now